src/replication/replication_schema_extract.hpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2008 Search Solution Corporation. All rights reserved by Search Solution.
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or
+ *   (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ */
+
+/*
+ * replication_schema_extract.hpp
+ */
+
+#ifndef _REPLICATION_SCHEMA_EXTRACT_HPP_
+#define _REPLICATION_SCHEMA_EXTRACT_HPP_
+
+#include "printer.hpp"
+
+namespace cubreplication
+{
+
+  class net_print_output : public print_output
```

Comment:
net_print_output seems like an useful tool fit for cubcomm module; it would have to be more generic (separating it of buffer type and replication_schema_extract).

we might consider adding net_print_output for develop.

Reply:
I think is preferable to move it in cubcomm as generic helper class only when we will need it.
A generic "net_print_sender" requires a level of multiplexing (above buffer_type : module_type).
It is hard to estimate now that we can use the same protocol for a future functionality.

Reply:
Alright, let's postpone such tool when we'll have another use case.